<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeThemePage</class>
 <widget class="QWidget" name="ChangeThemePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border-image  : url(':/ui/themes/default/dialog_page.png')
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_close">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
	border-image: url(':/ui/btn_close.png')
}
QToolButton:hover{
   border:2px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_theme_default">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   border-image:  url(':/ui/themes/default.jpg');
   border-radius:4px;
}
QLabel:hover{
   border:2px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_theme_love">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   border-image:  url(':/ui/themes/love.jpg');
   border-radius:4px;
}
QLabel:hover{
   border:2px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>改变主题</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
